ancillary_codi_tables.sql

```diff
@@ -168,7 +185,7 @@ CREATE TABLE CODI.REFERRAL
 	CHECK(REFERRAL_STATUS in ('A', 'D', 'NI', 'UN', 'OT')),
 	CHECK(REFERRAL_PRIOR_AUTH in ('Y', 'N', 'R', 'NI', 'UN', 'OT')),
 	PRIMARY KEY(REFERRALID),
-	FOREIGN KEY(SOURCE_PROVIDER_ID) REFERENCES CDM.PROVIDER (PROVIDERID),
+	FOREIGN KEY(SOURCE_PROVIDERID) REFERENCES CDM.PROVIDER (PROVIDERID),

```

Comment:
`SOURCE_PROVIDER_ID` should be changed to `SOURCE_PROVIDERID` on line 181 as well (I can't comment on that line specifically).